src/transformers/models/character_bert/tokenization_character_bert.py

```diff
@@ -0,0 +1,979 @@
+# coding=utf-8
+# Copyright 2023 The Google AI Language Team Authors and The HuggingFace Inc. team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tokenization classes for CharacterBert."""
+
+
+import collections
+import os
+import unicodedata
+import warnings
+from collections.abc import Mapping, Sized
+from typing import Dict, List, Optional, Tuple, Union
+
+import numpy as np
+
+from ...tokenization_utils import PreTrainedTokenizer, _is_control, _is_punctuation, _is_whitespace
+from ...tokenization_utils_base import LARGE_INTEGER, BatchEncoding, EncodedInput, TruncationStrategy
+from ...utils import (
+    PaddingStrategy,
+    TensorType,
+    is_tf_tensor,
+    is_torch_tensor,
+    logging,
+    to_py_obj,
+)
+
+
+logger = logging.get_logger(__name__)
+
+VOCAB_FILES_NAMES = {
+    "mlm_vocab_file": "mlm_vocab.txt",
+}
+
+PRETRAINED_VOCAB_FILES_MAP = {
+    "mlm_vocab_file": {
+        "helboukkouri/character-bert-base-uncased": "https://huggingface.co/helboukkouri/character-bert-base-uncased/resolve/main/mlm_vocab.txt",
+    }
+}
+
+PRETRAINED_POSITIONAL_EMBEDDINGS_SIZES = {
+    "helboukkouri/character-bert-base-uncased": 512,
+}
+
+PRETRAINED_INIT_CONFIGURATION = {
+    "helboukkouri/character-bert-base-uncased": {"max_word_length": 50, "do_lower_case": True},
+}
+
+PAD_CHARACTER_ID = 260
```

Comment:
```suggestion
```

Reply:
Hey! Could you please elaborate on why I should remove this?
I've used the same logic as for BERT: https://github.com/huggingface/transformers/blob/v4.38.2/src/transformers/models/bert/tokenization_bert.py#L31-L114

Reply:
This was removed in the latestmain!